v2/src/translations/en_US.json

```diff
@@ -115,6 +133,31 @@
             "addressName": "Add custom names to addresses in order to easily track them through ethVM"
         }
     },
+    "address": {
+        "circulatingSupply": "Circulating Supply",
+        "24hTradingVolume": "24h Trading Volume",
+        "blockRewards": "Block rewards",
+        "uncleRewards": "Uncle rewards",
+        "totalSupply": "Total Supply",
+        "24h": {
+            "high": "24h High",
+            "change": "24h Change",
+            "low": "24h low"
+        },
+        "placeholder": {
+            "search": "Search by hash or name"
+        },
+        "tagline": {
+            "noToken": "This address does not hold any tokens",
+            "noTransactionHistory": "This address does not have any token transfer history",
+            "noNFT": "This address does not have any NFTs",
```

Comment:
Hey lets remove  "noNFTAtAddress": "This address does not hold any NFTs" and replace "noNFTAtAddress" with "noNFT".